sleap/gui/dialogs/importvideos.py

```diff
@@ -525,6 +538,26 @@ def paint(self, painter, option, widget=None):
         pass
 
 
+class MessageWidget(QWidget):
+    """Widget to show message."""
+
+    def __init__(self, message: str = str(), *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.message = QLabel(message)
+        self.message.setStyleSheet("color: red")
+        self.layout = QVBoxLayout()
+        self.layout.addWidget(self.message)
+        self.setLayout(self.layout)
+
+    def boundingRect(self) -> QRectF:
+        """Method required by Qt."""
+        return QRectF()
+
+    def paint(self, painter, option, widget=None):
+        """Method required by Qt."""
+        pass
```

Comment:
Really?? I would think you wouldn't need to override it, or at least have a `super().paint(...)` in there?

Reply:
Admittedly, I meant to test this myself and had not gotten to it - will test before next review.

Reply:
It does not seem to be required by Qt.